packages/react-scripts/package.json

```diff
@@ -77,7 +77,7 @@
     "prompts": "2.4.0",
     "react-app-polyfill": "^2.0.0",
     "react-dev-utils": "^11.0.3",
-    "react-refresh": "^0.8.3",
+    "react-refresh": "^0.10.0",
```

Comment:
If there's a `^` it will only go to the latest version of the highest non-zero version number so `^0.8.3` will only ever go to the max `0.8.x` version of the package.

Reply:
Ah i didnt realize that. So for `0.x`, the `^` behaves like a `~`. Good to know, thanks